roseau/load_flow/models/lines/parameters.py

```diff
@@ -735,6 +740,151 @@ def from_name_mv(cls, name: str, max_current: float | Q_[float] | None = None) -
+    def from_open_dss(
+        cls,
+        id: Id,
+        *,
+        nphases: int,
+        r1: float | Q_[float],
+        r0: float | Q_[float],
+        x1: float | Q_[float],
+        x0: float | Q_[float],
+        c1: float | Q_[float],
+        c0: float | Q_[float],
+        basefreq: float | Q_[float] = F,
+        normamps: float | Q_[float] | None = None,
+        linetype: str | None = None,
+    ) -> Self:
+        """Create a line parameters object from OpenDSS "LineCode" data.
+
+        Args:
+            id:
+                The unique ID of the line parameters.
+
+            nphases:
+                OpenDSS parameter: `NPhases`. Number of phases in the line this line code represents.
+                To create single-phase lines with a neutral pass nphases=2, for three-phase lines with
+                a neutral nphases=4, etc.
+
+            r1:
+                OpenDSS parameter: `R1`. Positive-sequence resistance in (ohm/km).
+
+            r0:
+                OpenDSS parameter: `R0`. Positive-sequence resistance in (ohm/km).
+
+            x1:
+                OpenDSS parameter: `X1`. Positive-sequence reactance in (ohm/km).
+
+            x0:
+                OpenDSS parameter: `X0`. Positive-sequence reactance in (ohm/km).
+
+            c1:
+                OpenDSS parameter: `C1`. Positive-sequence capacitance in (nF/km).
+
+            c0:
+                OpenDSS parameter: `C0`. Positive-sequence capacitance in (nF/km).
+
+            basefreq:
+                OpenDSS parameter: `BaseFreq`. Frequency at which impedances are specified (Hz).
+                Defaults to 50 Hz.
+
+            normamps:
+                OpenDSS parameter: `NormAmps`. Normal ampere limit on line (A). This is the so-called
+                Planning Limit. It may also be the value above which load will have to be dropped
+                in a contingency. Usually about 75% - 80% of the emergency (one-hour) rating.
+                This value is passed to `max_current` and used for violation checks.
+
+            linetype:
+                OpenDSS parameter: `LineType`. Code designating the type of line. Only ``"OH"``
+                (overhead) and ``"UG"`` (underground) are currently supported.
+
+        Returns:
+            The corresponding line parameters object.
+
+        Example usage::
+
+            # DSS command: `New linecode.240sq nphases=3 R1=0.127 X1=0.072 R0=0.342 X0=0.089 units=km`
+            lp = LineParameters.from_open_dss(
+                id="linecode-240sq",
+                nphases=3,  #  creates 3x3 Z,Y matrices
+                r1=Q_(0.127, "ohm/km"),
+                x1=Q_(0.072, "ohm/km"),
+                r0=Q_(0.342, "ohm/km"),
+                x0=Q_(0.089, "ohm/km"),
+                c1=Q_(3.4, "nF/km"),  # default value used in OpenDSS code
+                c0=Q_(1.6, "nF/km"),  # default value used in OpenDSS code
```

Comment:
If they are the default values, they may be added in the signature of the function?